model/notification/payload.go

```diff
@@ -0,0 +1,29 @@
+package notification
+
+type EmailPayload struct {
+	Headers map[string]string `bson:"headers"`
+	Subject string            `bson:"subject"`
+	Body    []byte            `bson:"body"`
+}
+
+type GithubStatusAPIPayload struct {
+	Status      string `bson:"status"`
+	Context     string `bson:"context"`
+	Description string `bson:"description"`
+	URL         string `bson:"url"`
+}
+
+// TODO
+// This is a copy of message.JiraIssue, but with bson tags.
+// Is there a way around this?
```

Comment:
I'm fine putting bson tags in grip to avoid duplicating the type.

Reply:
PR: https://github.com/mongodb/grip/pull/24